Source/Scene/Expression.js

```diff
@@ -645,6 +664,13 @@ define([
         };
     }
 
+    function getEvaluateTernaryFunction(call) {
+        var evaluate = ternaryFunctions[call];
+        return function(feature) {
+            return evaluate(this._left.evaluate(feature));
```

Comment:
This needs to also evaluate `right` and `test`, like here: https://github.com/AnalyticalGraphicsInc/cesium/pull/4688/files#diff-113136b97bec6733c699e609394258e1R662